tests/autorestart/test_container_autorestart.py

```diff
@@ -71,7 +71,7 @@ def ignore_expected_loganalyzer_exception(loganalyzer):
         ".*WARNING syncd#syncd.*saiDiscover: skipping since it causes crash.*",
     ]
     teamd_ignoreRegex = [
-        ".*ERR teamd#teamsyncd.*readData.*netlink reports an error=-33 on reading a netlink socket.*",
+        ".*ERR swss#portsyncd.*readData.*netlink reports an error=-33 on reading a netlink socket.*",
```

Comment:
should we keep generic string and remove swss#portsyncd ?

Reply:
I considered that too. But I think we should be very selective in ignoring the ERR messages. So that we don't accidently ignore any real issues.